src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
So any change to any object property would be considered a breaking change? If someone is using `ReactWrapper.complexSelector.find` and we rename/move that, would that require a major release? Maybe we're not talking about the same thing. I usually define public API as the API that we document for the public and that we support.